server/cmd/start.go

```diff
@@ -36,13 +36,14 @@ var (
 	}
 )
 
-func Start(insecure bool, port int, namespaced bool, managedNamespace string) {
+// add accessPath to change base path for URL
+func Start(insecure bool, port int, namespaced bool, managedNamespace string, baseHref string) {
 	logger := logging.NewLogger().Named("server")
 	router := gin.New()
 	router.Use(gin.LoggerWithConfig(gin.LoggerConfig{SkipPaths: []string{"/livez"}}))
 	router.RedirectTrailingSlash = true
-	router.Use(static.Serve("/", static.LocalFile("./ui/build", true)))
-	routes.Routes(router, routes.SystemInfo{ManagedNamespace: managedNamespace, Namespaced: namespaced})
+	router.Use(static.Serve(baseHref, static.LocalFile("./ui/build", true)))
```

Comment:
Based on the code above, the `baseHref` is empty string by default, will that be a problem?